executor/include/executor/default/omp_executor.hpp

```diff
@@ -36,6 +36,16 @@ struct omp_executor {
     int idx;
   };
 
+  std::size_t max_threads;
```

Comment:
Use size_type please

Reply:
`size_type` of which container? I didn't quite follow you.

Reply:
Oops, my mistake. `shape_type`